_projects/mlcn.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Machine Learning for the Cognitive Neurosciences
+subtitle: Teaching workshop for applying ML methods to neuroscience
+status: active
```

Comment:
It should have a 'description' field as well !

Reply:
Done.